### ISSUE TITLE
[Usage]  finetune my data ，Why is the loss not decreasing?

### ISSUE THREAD

**Issue by awzhgw** [haotian-liu/LLaVA#1172]
### Describe the issue

when i finetune mydata with lora, the loss not decreasing ,why ,how to resolve it ?

![image](https://github.com/haotian-liu/LLaVA/assets/47099843/9f34d06b-cc18-4cc6-8235-cdf6f5bbd9fd)

**Comment by yuan10li20221130** [comment]
the training code for llava-v1.6-34b seems not open source yet,did you write the training code based on llava-v1.5?

**Comment by awzhgw** [comment]
any help ?

**Comment by yuan10li20221130** [comment]
Hi,maybe your learning rate is too large，if I were training your model,I would try constant learning rate 2e-5.
By the way,
Is your training code write by yourself or use llava-1.5 to train llava-v1.6-34b?

**Comment by awzhgw** [comment]
> Hi,maybe your learning rate is too large，if I were training your model,I would try constant learning rate 2e-5. By the way, Is your training code write by yourself or use llava-1.5 to train llava-v1.6-34b?

i use llava-1.5 to finetune my data. 
learing rate is 2e-5 

how to resolve it ?

**Comment by awzhgw** [comment]
> Hi,maybe your learning rate is too large，if I were training your model,I would try constant learning rate 2e-5. By the way, Is your training code write by yourself or use llava-1.5 to train llava-v1.6-34b?

my start shell is :
```shell
nohup deepspeed llava/train/train_mem.py \
    --lora_enable True --lora_r 128 --lora_alpha 256 --mm_projector_lr 2e-5 \
    --deepspeed ./scripts/zero3.json \
    --model_name_or_path ${CheckPoint_ROOT}/${ModelName} \
    --version v1 \
    --data_path ${JSON_PATH} \
    --image_folder ${IMAGE_FOLDER} \
    --vision_tower ${VISION_TOWER} \
    --mm_projector_type mlp2x_gelu \
    --mm_vision_select_layer -2 \
    --mm_use_im_start_end False \
    --mm_use_im_patch_token False \
    --image_aspect_ratio pad \
    --group_by_modality_length True \
    --bf16 True \
    --output_dir ${CheckPoint_ROOT}/${ModelName}-lora \
    --num_train_epochs 1 \
    --per_device_train_batch_size 16 \
    --per_device_eval_batch_size 4 \
    --gradient_accumulation_steps 1 \
    --evaluation_strategy "no" \
    --save_strategy "steps" \
    --save_steps 2500 \
    --save_total_limit 1 \
    --learning_rate 2e-5 \
    --weight_decay 0. \
    --warmup_ratio 0.03 \
    --lr_scheduler_type "cosine" \
    --logging_steps 1 \
    --tf32 True \
    --model_max_length 2048 \
    --gradient_checkpointing True \
    --dataloader_num_workers 1 \
    --lazy_preprocess True \
    --report_to tensorboard  > ${Local_DATA_ROOT}/llava_tune.log 2>&1 &
```

**Comment by tangjunjun966** [comment]
I met, how to solve

**Comment by OliverLeeXZ** [comment]
Same question! I also find that on finetine_lora.sh script provided by haotianliu learning rate is 2e-4, so we should change it to 2e-5?
